src/main/java/at/hannibal2/skyhanni/features/chat/Translator.kt

```diff
@@ -43,7 +43,7 @@ class Translator {
         style.setChatClickEvent(
             ClickEvent(
                 ClickEvent.Action.RUN_COMMAND,
-                "/shsendtranslation ${messageContentRegex.find(message)!!.groupValues[1]}"
+                "/shtranslate ${messageContentRegex.find(message)!!.groupValues[1]}"
```

Comment:
dont like the !! but it seems out of the scope of your pr and afaik hasnt caused an NPE for anyone yet so thats fine i guess

Reply:
Out of scope. Adding a `// TODO` is also not necessary, as we can just search for `!!` in the whole project.